react-elements/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
This is correct based on the documentation, but the better `import` for `ReactDOM` is:
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```

We'll cover in Q & A why that is.